sdk/face/ai-vision-face-rest/samples-dev/detect.ts

```diff
@@ -0,0 +1,73 @@
+import { readFileSync } from 'fs';
+import { AzureKeyCredential } from '@azure/core-auth';
+
+import createFaceClient, {
+    isUnexpected,
+} from '@azure-rest/ai-vision-face';
+
+/**
+ * This sample demonstrates how to create a liveness detection session.
+ *
+ * @summary creates a liveness detection session
+ */
+
+const detectFromImage = async () => {
+    const endpoint = process.env['FACE_ENDPOINT'] ?? '<endpoint>';
+    const apikey = process.env['FACE_APIKEY'] ?? '<apikey>';
+    const credential = new AzureKeyCredential(apikey);
+    const client = createFaceClient(endpoint, credential);
+
+    const filename = 'samples-dev/data/detection5.jpg';
```

Comment:
@jeremymeng do you know who could help on `samples publish` tool? how could we maintain the static files referred in samples?

@chungshengfu I would suggest we use relevative path like `data/detection5.jpg` to refer the file path. And then after run `npx dev-tool samples publish -f` we could manually copy the `data` folder to `samples/v1-beta/javascript` and typescript folder.